src/engine/ruleset/integrations/windows/manifest.yml

```diff
@@ -1,7 +1,12 @@
 name: integration/windows/0
 decoders:
 - decoder/windows-security/0
-- decoder/windows-powershell-operational/0
-- decoder/windows-event/0
-- decoder/windows-powershell/0
 - decoder/windows-sysmon/0
+- decoder/windows-powershell/0
```

Comment:
Could the list be sorted or does it require a specific order for some reason?

Reply:
in that order it is generated by the engine-integration script